src/main/java/com/hubspot/jinjava/JinjavaConfig.java

```diff
@@ -221,6 +227,7 @@ public ExecutionMode getExecutionMode() {
     private int maxMacroRecursionDepth;
     private boolean failOnUnknownTokens;
     private boolean nestedInterpretationEnabled = true;
+    private boolean usePyishObjectMapper = false;
```

Comment:
It is disabled by default, as it intentionally makes changes to the output. This may be changed to be enabled by default after a major release